nautobot/extras/models/customfields.py

```diff
@@ -161,6 +161,53 @@ def get_custom_fields(self, advanced_ui=None):
         # 2.0 TODO: #824 field.slug rather than field.name
         return OrderedDict([(field, self.cf.get(field.name)) for field in fields])
 
+    def get_custom_field_groupings_basic(self):
+        """
+        Return a dictonary of custom fields grouped by the same grouping in the form
+        {
+            <grouping_1>: [cf1, cf2...],
+            ...
+            <grouping_5>: [cf8, cf9...],
+            ...
+        }
+        which have advanced_ui set to False
+        """
+        return self.get_custom_field_groupings(advanced_ui=False)
+
+    def get_custom_field_groupings_advanced(self):
+        """
+        Return a dictonary of custom fields grouped by the same grouping in the form
+        {
+            <grouping_1>: [cf1, cf2...],
+            ...
+            <grouping_5>: [cf8, cf9...],
+            ...
+        }
+        which have advanced_ui set to True
+        """
+        return self.get_custom_field_groupings(advanced_ui=True)
```

Comment:
Can we get some notes in here as to why these methods need to exist where calling `get_custom_field_groups` with the properly defined `advanced_ui` kwarg is insufficient?

I assume this is due in part to the way they are being called in the template?